README.md

```diff
@@ -32,3 +32,26 @@ Use the Client
             .setClient(new MockClient(getContext(), "mocks"))
             .build();
 ```
+
+Notes
+----
+If you are unsure what to name your mocks, check the logs.  It will include both the expected name and the hash incase the filename is too long for your file system.
```

Comment:
heh, I also it should be "Because of this*" down below.  Definitely tired when I wrote this.